src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -283,6 +298,8 @@ private DocumentNode GenerateCosmosGraphQLObjects(HashSet<string> dataSourceName
                     case DatabaseType.CosmosDB_NoSQL:
                         cosmosDataSourceNames.Add(_runtimeConfigProvider.GetConfig().GetDataSourceNameFromEntityName(entityName));
                         break;
+                    case DatabaseType.DWSQL:
```

Comment:
is this needed? next case condition has DWSQL

Reply:
fixed.